ospfd/ospf_main.c

```diff
@@ -75,6 +76,7 @@ struct zebra_privs_t ospfd_privs = {
 const struct option longopts[] = {
 	{"instance", required_argument, NULL, 'n'},
 	{"apiserver", no_argument, NULL, 'a'},
+	{ "apiserver_addr", required_argument, NULL, 'l'},
```

Comment:
nit: additional whitespace at the beginning not needed (according to others above) :)

Reply:
Ok.